javascript/javascript1/week4/Homework/highest_and_lowest.js

```diff
@@ -0,0 +1,18 @@
+console.log("===========Highest and Lowest===========");
+
+function highAndLow(numbers) {
+  numbers = numbers.split(" ");
```

Comment:
```suggestion
  const numbersArray = numbers.split(" ");
```

It is recommended not to override your function's arguments.